_includes/get-started/create-first-app.html

```diff
@@ -10,10 +10,11 @@ <h1>Create your first application</h1>
 </p>
 
 <h3>Configuring your IDE</h3>
-<p>Currently the <a href="https://www.eclipse.org">Eclipse IDE</a> is the only IDE that has complete tooling support for both process and rule modeling.  As a result, we currently recommend using Eclipse IDE, but the steps below could also be followed with other IDEs.  You will however have to use an external tool for updating the process and/or rules.  The team is working on bringing in web-based embeddable editors (BPMN2, DMN and Scenario Simulation) to <a href="https://code.visualstudio.com/">Visual Studio Code</a> and <a href="https://eclipse.org/che">Eclipse Che</a> or <a href="https://developers.redhat.com/products/codeready-workspaces">Red Hat CodeReady Workspaces</a>.</p>
-
+<p>Currently the <a href="https://www.eclipse.org">Eclipse IDE</a> is the only IDE that has complete tooling support for both process and rule modeling, however Kogito Tooling provides a <a href="https://code.visualstudio.com/">Visual Studio Code</a> BPMN extension and the team is working on bringing in web-based embeddable editors (BPMN2, DMN and Scenario Simulation) to <a href="https://eclipse.org/che">Eclipse Che</a> or <a href="https://developers.redhat.com/products/codeready-workspaces">Red Hat CodeReady Workspaces</a>.</p>
+<h4>Install Visual Studio Code Kogito Tooling</h4>
+<p>
+Currently Kogito Tooling provides BPMN support (Alpha) for Visual Studio Code (VSCode). Currently it is not available in VSCode marketplace, so download the BPMN VSIX file from <a href="https://github.com/kiegroup/kogito-tooling/releases">Kogito Tooling releases page</a>, in VSCode select Extensions (<em>View -> Extensions</em> or use the shortcut in Linux/Windows systems <em>Ctrl + Shift + X</em>), then select the 'More Options' (three dots) icon and load the downloaded VSIX file.</p>
```

Comment:
can we reduce number of `Currently` being used... feels like its just too many times in these few sentences.

Reply:
Thank you, just updated my commit - let me know if there's anything you would to change